e2e/utils/create-project-utils.ts

```diff
@@ -81,7 +81,7 @@ export function newProject({
       packageInstall(packages.join(` `), projScope);
 
       // stop the daemon
-      execSync('nx reset', {
+      execSync('npx nx reset', {
```

Comment:
should'd we use the `packageManaer` instead?

Reply:
We could but it's not necessary in this case. This should happen only once per test project run and ensures daemon is stopped before we copy workspace to another folder.